bindings/node.js/kzg.cxx

```diff
@@ -9,8 +9,8 @@
 #include "blst.h"
 
 Napi::Value throw_invalid_arguments_count(
-  const uint expected,
-  const uint actual,
+  const unsigned int expected,
+  const unsigned int actual,
```

Comment:
Fixes the EthereumJS issue https://github.com/ethereumjs/ethereumjs-monorepo/pull/2349#issuecomment-1322197299